README.md

```diff
@@ -1,24 +1,126 @@
-# README
+<a name="readme-top"></a>
 
```

Comment:
- [ ] You have not implemented any of the migrations needed for this project.

  ![ireo](https://user-images.githubusercontent.com/54848363/216291192-58509eab-a9bf-414c-a655-be98d4be532d.jpg)

  Kindly follow the needed lessons and run the migrations for generating the different models: `User`, `Post`, `Comment`, and `Like`. The tables should have the columns shared with you in the ERD diagram.